include/wayland-client-core.h

```diff
@@ -0,0 +1,323 @@
+/*
```

Comment:
It would be better to download these files using the package manager. This makes it also clearer that these come from the official source, which is something I'd otherwise need to verify manually.

Reply:
I think it would make more sense to do this in a separate PR given the other vendored headers in this repo. It also isn't obvious at all how this should be done. Could be something we do manually in build.zig, a separate repo we fetch in build.zig.zon and so on. I took the headers from the Nix package, so they should be identical to the upstream code. Feel free to diff them.